driver-core/src/main/com/mongodb/internal/operation/ClientBulkWriteOperation.java

```diff
@@ -315,6 +512,25 @@ private <R> R doWithRetriesDisabledForCommand(
         }
     }
 
+    private <R> void doWithRetriesDisabledForCommandAsync(
+            final RetryState retryState,
+            final String commandDescription,
+            final AsyncSupplier<R> actionWithCommand,
+            final SingleResultCallback<R> finalCallback) {
+        Optional<Boolean> originalRetryableCommandFlag = retryState.attachment(AttachmentKeys.retryableCommandFlag());
+        Supplier<String> originalCommandDescriptionSupplier = retryState.attachment(AttachmentKeys.commandDescriptionSupplier())
+                .orElseThrow(Assertions::fail);
+
```

Comment:
As discussed on the call, instead of removing the line breaks from the async implementation, I’ve added the same line breaks to the sync version for consistency.